tools/xtce_generation/telemetry_generator.py

```diff
@@ -212,7 +231,9 @@ def create_ccsds_packet_parameters(self, parameter_set, ccsds_parameters):
             The updated ParameterSet element.
         """
         for parameter_data in ccsds_parameters:
-            parameter = Et.SubElement(parameter_set, "xtce:Parameter")
+            parameter = Et.SubElement(parameter_set, "xtce:Parameter")  # type: ignore[arg-type]
+            # Argument 1 to "SubElement" has incompatible type
+            # "list[str]"; expected "Element"
```

Comment:
```suggestion
            parameter = Et.SubElement(parameter_set, "xtce:Parameter")
```

Should be able to remove these with the above fix